Content.Server/Chemistry/ReagentEffects/PlantMetabolism/PlantClonexadone.cs

```diff
@@ -1,19 +1,21 @@
 ﻿using System;
 using Content.Server.Botany.Components;
 using Content.Shared.Botany;
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Chemistry.Reagent;
 using JetBrains.Annotations;
 using Robust.Shared.GameObjects;
 using Robust.Shared.IoC;
 using Robust.Shared.Random;
 using Robust.Shared.Serialization.Manager.Attributes;
 
-namespace Content.Server.Chemistry.PlantMetabolism
+namespace Content.Server.Chemistry.ReagentEffects.PlantMetabolism
 {
     [UsedImplicitly]
     [DataDefinition]
-    public class Clonexadone : IPlantMetabolizable
+    public class PlantClonexadone : ReagentEffect
     {
-        public void Metabolize(IEntity plantHolder, float customPlantMetabolism = 1)
+        public override void Metabolize(IEntity plantHolder, IEntity organEntity, Solution.ReagentQuantity amount)
```

Comment:
You can import static members of Solution here (and in other places) to make the type shorter.

Reply:
i can technically do that but the import is definitely longer than the type name is